src/test/assembly/x86_64-floating-point-clamp.rs

```diff
@@ -0,0 +1,25 @@
+// Floating-point clamp is designed to be implementable as max+min,
+// so check to make sure that's what it's actually emitting.
+
+// assembly-output: emit-asm
+// compile-flags: --crate-type=lib -O -C llvm-args=-x86-asm-syntax=intel
+// only-x86_64
+
+// CHECK-LABEL: clamp_demo:
+#[no_mangle]
+pub fn clamp_demo(a: f32, x: f32, y: f32) -> f32 {
+    // CHECK: maxss
+    // CHECK: minss
+    a.clamp(x, y)
+}
+
+// CHECK-LABEL: clamp12_demo:
+#[no_mangle]
+pub fn clamp12_demo(a: f32) -> f32 {
+    // CHECK-NEXT: movss   xmm1
```

Comment:
Sigh, I have no idea why CI gets
```
        38: clamp12_demo: 
        39:  .cfi_startproc 
        40:  movss xmm1, dword ptr [rip + .LCPI1_0] 
next:19      !~~~~~~~~~                              error: match on wrong line
        41:  maxss xmm1, xmm0 
        42:  movss xmm0, dword ptr [rip + .LCPI1_1] 
        43:  minss xmm0, xmm1 
        44:  ret 
        45: .Lfunc_end1: 
```
since I just get
![image](https://user-images.githubusercontent.com/18526288/185006707-46fd9eb2-5414-4fd9-ba92-d4a07b11dfd9.png)

Probably the best answer is to downgrade the precision of this check like
```suggestion
    // CHECK: movss   xmm1
```